crates/network/src/any/mod.rs

```diff
@@ -74,3 +76,39 @@ impl Network for AnyNetwork {
 
     type BlockResponse = AnyRpcBlock;
 }
+
+/// A wrapper for [`AnyRpcBlock`] that allows for handling unknown block types.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct AnyRpcBlockWrapper(pub AnyRpcBlock);
```

Comment:
in order to make this non breaking we need to rename this struct to `AnyRpcBlock`

we also need to expose all the functions that WithOtherFields has so that becomes non breaking

Reply:
remove the alias and place it where `pub AnyRpcBlock` is